pom.xml

```diff
@@ -67,14 +67,9 @@
 
 	<dependencies>
 		<dependency>
-			<groupId>org.eclipse.jetty</groupId>
-			<artifactId>jetty-server</artifactId>
-			<version>8.1.8.v20121106</version>
-		</dependency>
```

Comment:
I include every explicit dependency (that are used directly by the code of the project), no implicit (due to transitive). If the code use class from jetty-server, then list it as a dependency.